scripts/import/import_clinvar_xml

```diff
@@ -589,6 +613,15 @@ sub import{
   if(defined $record->{feature_info}->{dbSNP} && !defined $structvar){
     foreach my $rs (@{ $record->{feature_info}->{dbSNP} }){
 
+      # for haplotypes (multiple rsIDs) each rsID has it's own hgvs_g string
+      if (defined $record->{feature_info}->{'rs'.$rs}){
```

Comment:
We could use a global variable `my $rsid = 'rs'.$rs;` and/or assign:
```
my $record_feature = $record->{feature_info};
my $record_feature_info = $record_feature->{$rsid};

$record_feature->{hgvs_g} = $record_feature_info->{hgvs_g};
...
```

Reply:
Thanks for the suggestion. Added it in the next commit.